draft-ietf-quic-recovery.md

```diff
@@ -372,10 +366,17 @@ one significantly increases resilience to packet drop in both directions, thus
 reducing the probability of consecutive RTO events.
 
 QUIC's RTO algorithm differs from TCP in that the firing of an RTO alarm is not
-considered a strong enough signal of packet loss. An RTO alarm fires only when
+considered a strong enough signal of packet loss, so does not result in an
+immediate change to CWND or recovery state. An RTO alarm fires only when
```

Comment:
is CWND defined by here? or should it be "congestion window"? I'd prefer spelling it out in any case.

Reply:
Done